question-service/controller/questions-controller.js

```diff
@@ -0,0 +1,46 @@
+// Import contact model
+import { ormGetRandomQuestion as _getRandomQuestion, ormGetQuestionByDifficulty as _getQuestionByDifficulty, ormGetQuestionByTopic as _getQuestionByTopic, ormGetTopics as _getTopics } from '../model/questions-orm.js';
+
+
+export async function viewRandomQuestion(req, res) {
+    try {
+        const question = await _getRandomQuestion();
+        return res.status(200).json({ question: question });
+    } catch (err) {
+        res.status(500).json({ message: 'Database failure when getting random question'});
```

Comment:
Should there be a `return` keyword here? Same question for the other methods in this controller.